app/services/dashboard-stats.js

```diff
@@ -350,15 +350,15 @@ export default class DashboardStatsService extends Service {
             }, 0);
         }
 
-        yield this.loadPaidProducts();
+        yield this.loadPaidTiers();
 
         const paidMembersByTier = [];
 
         for (const tier of result.meta.tiers) {
-            const product = this.paidProducts.find(x => x.id === tier);
+            const _tier = this.paidTiers.find(x => x.id === tier);
```

Comment:
Not sure why product and tier were being used interchangeably here, but I had to rename this to `_tier` as there's already a `tier` variable 🤔